src/cmd/mod.rs

```diff
@@ -491,10 +532,10 @@ pub fn display_psbt(network: Network, psbt: &Psbt) -> String {
         "total".into(),
         format_amount(output_total),
     ]));
-    let (fee, feerate) = psbt.fee();
+    let (fee, feerate, fee_estimated) = psbt.fee();
 
     table.add_row(Row::new(vec![
-        "fee",
+        if fee_estimated { "est. fee" } else { "fee" },
```

Comment:
It's not the fee that's being estimated here it's the feerate that was estimated.

Reply:
I think put `3.2 sat/vb (est.)` for example